### DISCUSSION TITLE
Scraping within a webpage?

### DISCUSSION THREAD

**Discussion by ReporterMarina** [bertrandmartel/tableau-scraping#42] [Q&A]
Hi, any advice for getting data from an embedded Tableau table?

https://coronavirus.health.ny.gov/daily-hospitalization-summary

I used to be able to go to the Tableau URL for that table, but it seems like it doesn't exist anymore: 

https://covid19tracker.health.ny.gov/views/DailyHospitalizationSummary/Reopening-DailyHospitalization?%3Aembed=y&%3AshowVizHome=n&%3Atabs=n&%3Atoolbar=yes&%3Adevice=desktop&%3AapiID=host0#navType=1&navSrc=Parse

**Comment by bertrandmartel** [comment]
The URL for the first link is: https://public.tableau.com/views/DailyHospitalizationSummary_Public/Reopening-DailyHospitalization

For the second link, the target url is: https://public.tableau.com/views/NYS-COVID19-Tracker_Public/NYSDOHCOVID-19Tracker-Map
